css/styles.css

```diff
@@ -0,0 +1,240 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  width: 1200px;
+  margin: 0 auto;
+  overflow: auto;
+}
+
+body {
+  background-color: darkgray;
+}
+
+header {
+  height: 145px;
+  background-color: black;
+  border: 1px solid #333;
+  margin: 0px;
+  color: white;
+}
+
+h1 {
+  color: yellow;
+  padding-left: 20px;
+  font-size: 50px;
+  margin: 1px 10px 1px 10px;
+  padding: 0px 10px 0px 10px;
+  font-family:
+
+}
+nav {
+  margin: 1px 10px 1px 10px;
+  padding: 1px 10px 1px 10px;
+  text-align: right;
+  color: white;
+  font-family: Georgia, serif;
+  font-size: 20px;
+}
+ul a {
+  list-style-type: none;
+  margin: 1px 5px 1px 5px;
+  padding: 1px 5px 1px 5px;
+  text-align: right;
+  color: white;
+  font-family: Georgia, serif;
+  font-size: 25px;
+  text-decoration: none;
+}
+
+li {
+    display: inline;
+}
+
+nav span {
+  text-transform: uppercase;
+  font-weight: bold;
+
+}
+
+section {
+  height: 100px;
+  background-color: lightgray;
+  border: 5px solid transparent;
+  margin: 0px;
+  padding: 20px;
+  font-family: sans-serif;
+  text-align: center;
+  font-size: 40px;
+  border-top: solid yellow;
+}
+
+article {
+  height: 500px;
+  border: 1px solid #333;
+  width: 100%;
+  background-image: url("../images/coworking.jpg");
+  background-repeat: no-repeat;
+  background-position: center center;
```

Comment:
@aaronbatsss this could've used the short hand of background `background: url("../images/coworking.jpg") no-repeat center center;`

Reply:
@rmjames Hi Robert, would you like me to change it to reflect that short hand?

Reply:
@aaronbatsss No, only make changes and corrections if you see a 🔨 . I'm just suggesting a different solution that you might find useful.

Reply:
@rmjames Fair enough, appreciate it.